src/test/java/com/exasol/adapter/document/files/AdlsRemoteFileContentTest.java

```diff
@@ -0,0 +1,78 @@
+package com.exasol.adapter.document.files;
+
+import com.exasol.adapter.document.documentfetcher.files.RemoteFileContent;
+import com.exasol.adapter.document.files.adlstestsetup.AdlsTestSetup;
+import com.exasol.adapter.document.files.adlstestsetup.OnlineAdlsTestSetup;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class AdlsRemoteFileContentTest {
```

Comment:
This looks like an integration test, please rename the class to `...IT`

Reply:
makes sense, done